src/test/java/co/rsk/federate/signing/hsm/client/PowHSMSigningClientBtcTest.java

```diff
@@ -84,7 +85,7 @@
 
 class PowHSMSigningClientBtcTest {
 
-    private static final int HSM_VERSION_2 = 2;
+    private static final HSMVersion HSM_VERSION_2 = HSMVersion.V2;
```

Comment:
Remove this? Use the enum directly as in other places

Reply:
Good catch. I missed this one.